types/react-select/index.d.ts

```diff
@@ -3,13 +3,18 @@
 // Definitions by: Claas Ahlrichs <https://github.com/claasahl>
 //                 Jon Freedman <https://github.com/jonfreedman>
 //                 Nathan Bierema <https://github.com/Methuselah96>
+//                 Thomas Chia <https://github.com/thchia>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.9
 
 import { StateManager } from './src/stateManager';
 
 export default StateManager;
 
+export { Props as SelectProps } from './src/Select';
+export { Props as AsyncProps } from './src/Async';
+export { Props as CreatableProps } from './src/Creatable';
+export { Props as AsyncCreatableProps } from './src/AsyncCreatable';
```

Comment:
I'm not in favor of exporting these from `index.d.ts` since the Flow types in the original repo aren't exported like that. Can't you import them via `import { Props } from 'react-select/src/Select`?

Reply:
This makes total sense. I'll update this to just export the `Props` from `AsyncCreatable` then.